app/assets/stylesheets/rat_adoptions.css.scss

```diff
@@ -0,0 +1,90 @@
+.stats {
+  @include span-columns(12);
+  @include shift(3.5);
+  $base-spacing: 1.5em !default;
+  padding: base-spacing /Z;
+  margin-top: 50px;
+  margin-bottom: 40px;
+  text-align: center;
+
+  ul {
+    padding: 0;
+  }
+
+  li {
+    border-right: 1px solid transparentize($dark-blue, 0.8);
+    color: $blue;
+    display: inline;
+    float: left;
+    font-size: 1.2em;
+    line-height: 1.1em;
+    padding: 0 0.7em;
+
+    &:first-child {
+      padding-left: 0;
+    }
+
+    &:last-child {
+      border-right: 0;
+    }
+  }
+
+  span {
+    color: $dark-blue;
+    display: block;
+    font-size: 1em;
+    font-weight: normal;
+  }
+}
+
+.list-of-rats {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  
+  li {
+    display: inline-block;
+    height: 150px;
+    margin: 100px 1em 1em 0;
+    position: relative;
+    width: 150px;
+  }
+}
+
+.rat-name-container {
+  background: rgba(0, 0, 0, 0.5);
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets